pkg/cmd/render/render.go

```diff
@@ -192,6 +202,85 @@ func (r *renderOpts) Run() error {
 	return genericrender.WriteFiles(&r.generic, &renderConfig.FileConfig, renderConfig)
 }
 
+func getDefaultConfigWithAuditPolicy() ([]byte, error) {
+	defaultPolicy, err := audit.GetDefaultPolicy()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get default audit policy - %s", err)
+	}
+
+	policy, err := convertToUnstructured(defaultPolicy)
+	if err != nil {
+		return nil, fmt.Errorf("failed to decode audit policy into unstructured - %s", err)
+	}
+
+	asset := filepath.Join(bootstrapVersion, "config", "defaultconfig.yaml")
+	raw, err := v410_00_assets.Asset(asset)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get default config asset asset=%s - %s", asset, err)
+	}
+
+	rawJSON, err := kyaml.ToJSON(raw)
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert asset yaml to JSON asset=%s - %s", asset, err)
+	}
+
+	decoder := json.NewDecoder(bytes.NewBuffer(rawJSON))
+	defaultConfig := map[string]interface{}{}
+	if err := decoder.Decode(&defaultConfig); err != nil {
+		return nil, fmt.Errorf("failed to decode default config - %s", err)
+	}
+
+	if err := addAuditPolicyToConfig(defaultConfig, policy); err != nil {
+		return nil, fmt.Errorf("failed to add audit policy into default config - %s", err)
+	}
+
+	defaultConfigRaw, err := json.Marshal(defaultConfig)
+	if err != nil {
+		return nil, fmt.Errorf("failed to marshal default config - %s", err)
+	}
+
+	return defaultConfigRaw, nil
+}
+
+func addAuditPolicyToConfig(config, policy map[string]interface{}) error {
+	const (
+		auditConfigPath  = "auditConfig"
+		localAuditPolicy = "openshift.local.audit/policy.yaml"
+	)
+
+	auditConfigEnabledPath := []string{auditConfigPath, "enabled"}
```

Comment:
we have an override file for the config that is used for bootstrapping. You could hard code those there.

Reply:
I just wanted to avoid have part of it hard coded and then the rest being wired on the fly, Previously, we had the whole thing hard coded.